README.md

```diff
@@ -1,54 +1,36 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# 
+![](https://img.shields.io/badge/Weather-App-blueviolet)
 
-#### 
+# Fabulous-tours
 
+> The Weather app is a simple weather application that helps users to know the current weather status in Celsius as well as Fahrenheit.
 
-## Built with
+![screenshot](app.png)
 
-- Ruby -v 3.0.0
-- Rails -v 6.1.3.1
-- SQLite3
-- VScode
-- Lots of love :heart:
-
-## Getting Started
+## Built With
 
-### Prerequisites
+- HTML
+- Bootstrap
+- JavaScript
+- Open Weather API
+- Lots of love :heart:
 
-To get this project up and running locally, you must already have ruby and the necessary gems installed on your computer
+## Live Demo
 
```

Comment:
Kindly add instructions about how to clone and run the project locally in the README file :+1: